test/todos.js

```diff
@@ -197,4 +197,18 @@ describe(Todos, function() {
       actual.should.eql([2, 1]);
     });
   });
+
+  describe('#renumber', function() {
+    it('renumbers the todo items', function() {
+      var todos = new Todos(storage);
+
+      time.freeze();
```

Comment:
Good question. So I saw it used elsewhere when testing the what storage was called with:

```
storage.write.should.have.been.called.with.args([ new Todo(1, 'desc', 'done') ]);
```

I assumed it was because when creating a new `Todo`, there is a `modified` property that is set to `new Date()` and it *might* not match what was saved.

I didn't spend too much time looking into it, so if I'm misusing/misunderstanding, let me know and I'll remove it.